solutions/noisg-19-feast.mdx

```diff
@@ -0,0 +1,93 @@
+---
+id: noisg-19-feast
+source: NOI
```

Comment:
Also this should be something like "Singapore NOI" or "NOI.sg" to avoid confusion